app/presenters/dates_report_presenter.rb

```diff
@@ -0,0 +1,40 @@
+class DatesReportPresenter < CSVPresenter
+  def initialize(start_date, end_date)
+    start_date = Date.parse(start_date).beginning_of_day
```

Comment:
You're double-parsing here - the rake task already parses a string into a DateTime so when you run the rake task you get 
```
TypeError: no implicit conversion of Date into String
/var/govuk/publisher/app/presenters/dates_report_presenter.rb:3:in `parse'
```